src/Microsoft.AspNetCore.Authentication.OAuth/DeleteClaimMapper.cs

```diff
@@ -0,0 +1,33 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Linq;
+using System.Security.Claims;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.AspNetCore.Authentication
+{
+    /// <summary>
+    /// A JsonClaimMapper that deletes all claims from the given ClaimsIdentity with the given ClaimType.
+    /// </summary>
+    public class DeleteClaimMapper : JsonClaimMapper
```

Comment:
A delete mapper makes me think we should choose a different word.

like ClaimAction/Operation? with Map/Delete becoming something paired?
Copy/Delete or Add/Remove

```
Claims.CopyJsonKey(ClaimTypes.NameIdentifier, "id");
Claims.Delete("ClaimTypeToDelete");
```